docs/src/containers/WidgetForm.tsx

```diff
@@ -0,0 +1,210 @@
+import {
+  Button,
+  Dialog,
+  IconButton,
+  RadioButton,
+  RadioGroup,
+  TextField,
+} from "@statflo/ui";
+import { Formik } from "formik";
+import { useEffect, useState } from "react";
+import * as Yup from "yup";
+import { faker } from "@faker-js/faker";
+
+import { useWidgetContext } from "../providers/WidgetProvider";
+
+const WidgetSchema = Yup.object().shape({
+  name: Yup.string().required(),
+  label: Yup.string(),
+  url: Yup.string().required(),
+  position: Yup.string().oneOf(["sendable", "right_panel"]),
+  priorty: Yup.number(),
+});
+
+const initialValues: {
+  name: string;
+  label: string;
+  url: string;
+  position: "sendable" | "right_panel";
+  priority: number;
+} = {
+  name: "",
+  label: "",
+  url: "",
+  position: "right_panel",
+  priority: 99,
+};
+
+type WidgetFormProps = {
+  isEdit?: boolean;
+  widget?: Widget;
+};
+
+const WidgetForm = ({ isEdit, widget }: WidgetFormProps) => {
```

Comment:
Here you simulate the widget admin? I know you said you were attempting this, very nice.

Reply:
Yeah! It's basically a dumbed down version of widget admin so that it simulates what would actually be going on in the application.